src/test/java/org/openrewrite/staticanalysis/AddSerialVersionUidToSerializableTest.java

```diff
@@ -57,6 +57,34 @@ public class Example implements Serializable {
         );
     }
 
+    @DocumentExample
```

Comment:
We try to only have a single example for our docs, as these would be very similar.
```suggestion
```